src/mame/drivers/nmk16.cpp

```diff
@@ -1291,6 +1291,12 @@ void nmk16_state::bjtwin_map(address_map &map)
 	map(0x0f0000, 0x0fffff).ram().share("mainram");
 }
 
+void nmk16_state::sabotenb_map(address_map &map)
+{
+	bjtwin_map(map);
+	map(0x094003, 0x094003).w(FUNC(nmk16_state::sabotenb_scroll_w));    // sabotenb specific?
```

Comment:
I highly doubt Saboten Bombers uses scrolly any different than Bombjack Twin, more likely they have the same meaning and the two memory maps can be merged (and the note for bjtwin at $94002/3 being wrong).